app/src/org/commcare/dalvik/application/CommCareApp.java

```diff
@@ -131,8 +132,29 @@ public void setupSandbox(boolean createFilePaths) {
         }
     }
     
+    public void updateAppRecord() {

```

Comment:
this method should have documentation that outlines what it accomplishes, when it fires, etc.